tableauserverclient/models/site_item.py

```diff
@@ -125,118 +208,129 @@ def subscribe_others_enabled(self, value):
+            if 'numViewers' in usage_fields:
+                self._num_viewers = int(usage_fields['numViewers'])
+            if 'storage' in usage_fields:
+                self._storage = int(usage_fields['storage'])
 
     @classmethod
     def from_response(cls, resp, ns):
         all_site_items = list()
         parsed_response = ET.fromstring(resp)
         all_site_xml = parsed_response.findall('.//t:site', namespaces=ns)
         for site_xml in all_site_xml:
-            (id, name, content_url, status_reason, admin_mode, state, subscribe_others_enabled,
-                disable_subscriptions, revision_history_enabled, user_quota, storage_quota,
-                revision_limit, num_users, storage, materialized_views_mode) = cls._parse_element(site_xml, ns)
-
-            site_item = cls(name, content_url)
-            site_item._set_values(id, name, content_url, status_reason, admin_mode, state,
-                                  subscribe_others_enabled, disable_subscriptions, revision_history_enabled,
-                                  user_quota, storage_quota, revision_limit, num_users, storage,
-                                  materialized_views_mode)
+            site_fields = cls._parse_element(site_xml, ns)
+            site_item = cls(site_fields['name'], site_fields['contentUrl'])
+            site_item._set_values(site_fields)
             all_site_items.append(site_item)
         return all_site_items
 
     @staticmethod
     def _parse_element(site_xml, ns):
-        id = site_xml.get('id', None)
-        name = site_xml.get('name', None)
-        content_url = site_xml.get('contentUrl', None)
-        status_reason = site_xml.get('statusReason', None)
-        admin_mode = site_xml.get('adminMode', None)
-        state = site_xml.get('state', None)
-        subscribe_others_enabled = string_to_bool(site_xml.get('subscribeOthersEnabled', ''))
-        disable_subscriptions = string_to_bool(site_xml.get('disableSubscriptions', ''))
-        revision_history_enabled = string_to_bool(site_xml.get('revisionHistoryEnabled', ''))
-
-        user_quota = site_xml.get('userQuota', None)
-        if user_quota:
-            user_quota = int(user_quota)
-
-        storage_quota = site_xml.get('storageQuota', None)
-        if storage_quota:
-            storage_quota = int(storage_quota)
-
-        revision_limit = site_xml.get('revisionLimit', None)
-        if revision_limit:
-            revision_limit = int(revision_limit)
-
-        num_users = None
-        storage = None
+        site_fields = site_xml.attrib
         usage_elem = site_xml.find('.//t:usage', namespaces=ns)
         if usage_elem is not None:
-            num_users = usage_elem.get('numUsers', None)
-            storage = usage_elem.get('storage', None)
-
-        materialized_views_mode = site_xml.get('materializedViewsMode', '')
+            usage_fields = usage_elem.attrib
+            site_fields['usage'] = usage_fields
```

Comment:
This would conflict with an attribute named 'usage' if we were ever to add one to the site element. 

Would it be a little cleaner to return two dicts (or, in the case of more nested elements, one dict for each element) or maybe a wrapper dict that has a 'site_fields' and 'usage_fields' sub dict? It just feels like a cleaner model if we wanted to go apply this cleanup to other more deeply nested return types.

(And yeah, that means I'm coming around on just using 'attrib')